shared/variables.tf

```diff
@@ -28,4 +28,10 @@ locals {
   }
 
   s3_bucket = local.account.name == "production" ? "${local.account.name}02" : local.account.name
+
+  cloud9_users = [
+    "alex.saunders",
+    "gugandeep.chani",
+    "stephen.hinett",
```

Comment:
Could we add Mia in here? Probably worth having you in there too @xaithe if you need to do any DB work in the future.

Reply:
Think Tom will have access as he has set himself to Owner.